app/test/ui/widgets/custom_card_test.dart

```diff
@@ -0,0 +1,27 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:medi_support/ui/widgets/custom_card.dart';
+
+void main() {
+  CustomCard customCard = const CustomCard(
+    contentPadding: EdgeInsets.all(16),
+    child: Text('Test'),
+  );
+
+  testWidgets('CustomCard renders correctly', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: customCard,
+        ),
+      ),
+    );
+
+    expect(find.byType(CustomCard), findsOneWidget);
```

Comment:
**suggestion:** When testing the widget you dont need to check if its there just check for actual properties.